src/lib.rs

```diff
@@ -1,4 +1,8 @@
 //! A Rust port of R's C Library of Special Functions.
+
+// Avoid converting `if n < 1 || n > 1000` to `if !(1..=1000).contains(&n)`.
+#![allow(clippy::manual_range_contains)]
+
```

Comment:
Is this only happening on `chebyshev` for now?
Why don't we move this to the specific `chebyshev` function that it applies to?

Reply:
Because it can silently mess up our 1 on 1 conversion (if you just use clippy --fix)